docs/contributing.md

```diff
@@ -9,8 +9,13 @@ Things you can help with:
  - [Issues](#issues)
      - [Squashing bugs](#quick-links)
      - [Enhance opsdroid](#quick-links)
+<<<<<<< HEAD
```

Comment:
This is showing another conflict, check the difference between the two and go with the right one